src/components/reactTasks/dislike.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default class Dislike extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {subscribed: true}
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick() {
+    this.setState(prevState => ({
```

Comment:
This method of accessing state with the `prevState` argument is a little bit long winded, I would just use `this.state`